.github/workflows/demo/docker-compose.demo.yml

```diff
@@ -0,0 +1,10 @@
+version: "3"
+services:
+  haystack-api:
```

Comment:
In the base `docker-compose-gpu.yml`, I would like to change the command  line of the `haystack-api` container (to raise the number of workers). Is it possible to override it by specifying a `command` line here?

If yes, the line to add looks like:
`command: "/bin/bash -c 'sleep 10 && gunicorn rest_api.application:app -b 0.0.0.0 -k uvicorn.workers.UvicornWorker --workers 3 --timeout 180'"`